File: proyectoBienestar/app/Area.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Area extends Model
{
    protected $table = 'areas';
    protected $fillable = [
        'name', 'area_image', 'area_presentation', 'objetive', 'programs',
    ];

    public function users()
    {
    	return $this->hasMany('App\User');
    }

    public function activities()
    {
    	return $this->hasMany('App\Activity');
    }

    public function publications()
    {
    	return $this->hasMany('App\Publication');
    }
}

File: Android Bienestar/app/src/main/java/com/esteban/bienestarjdc/data/Area.kt
package com.esteban.bienestarjdc.data

data class Area(
    val areaActivities: List<AreaActivity>?,
    val area_image: String,
    val area_presentation: String,
    val id: Int,
    val name: String,
    val objetive: String,
    val programs: String,
    val publications: List<Publication>?,
    val users: List<User>?
)